mitodl/mitxonline
# Populate max min weekly hours

### What are the relevant tickets?
Fix https://github.com/mitodl/hq/issues/6099

### Description (What does it do?)
migrate existing "Effort" strings to weekly_hours integer fields


### How can this be tested?
Run the migration and make sure that max_weekly_hours and min_weekly_hours is showing the correct values.
